google/cloud/security/common/data_access/csv_writer.py

```diff
@@ -116,6 +116,20 @@
     'buckets'
 ]
 
+BUCKETS_ACL_FIELDNAMES = [
+    'bucket',
+    'domain',
+    'email',
+    'entity',
+    'entity_id',
+    'acl_id',
+    'kind',
+    'project_team',  # TODO: flatten this
+    'role',
+    'bucket_acl_selflink',
+    'raw_bucket_acl'
+]
+
 CSV_FIELDNAME_MAP = {
```

Comment:
nit: Please leave a TODO that we should start to alphabetize these entries so they're easier to find.